README.md

```diff
@@ -1 +1,31 @@
-# fastify-express
\ No newline at end of file
+# fastify-express
+
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)
+
+This plugin adds full [Express](http://expressjs.com) compatibility to Fastify, it exposes the same `use` function of Express, and it allows you to use any Express middleware.<br/>
```

Comment:
This plugin doesn't cover express `Router` functionality which is a commonly feature to encapsulate middlewares.

Reply:
Perhaps it should be possible to pass your own express isntance?

Reply:
> This plugin doesn't cover express `Router` functionality which is a commonly feature to encapsulate middlewar

Now, this feature is not supported (I read only the docs [here](https://github.com/fastify/fastify/blob/master/docs/Middlewares.md#express-middleware-compatibility) ). 
Do we want to support it?

Reply:
Then we should point to this compatibility doc.

Reply:
> This plugin doesn't cover express Router functionality which is a commonly feature to encapsulate middlewares.

Not sure how this could be implemented, I fear that will cause some weird bugs with our encapsulation system.
Have you got any suggestion?

Reply:
I think `express.Router` support is critical. Why do you think it's not supported?

Reply:
From what I can see, this plugin already supports `express.Router`. This code should already work:

```js
const Fastify = require('fastify')
const express = require('express')

const fastify = Fastify()
const router = express.Router()

router.use((req, res, next) => {
  console.log('Router middleware called')
  next()
})

fastify.register(require('fastify-express'))
  .after(() => {
    fastify.use(router)
  })
```